debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -343,7 +343,7 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                  .withDescription("Unique name that identifies the database server and all recorded offsets, and"
                                                          + "that is used as a prefix for all schemas and topics. "
                                                          + "Each distinct MySQL installation should have a separate namespace and monitored by "
-                                                         + "at most one Debezium connector. Defaults to 'host:port'");
+                                                         + "at most one Debezium connector.");
```

Comment:
Okay, I guess I'm fine with the docs not mentioning the default and instead keeping it as a required field. This is used as a prefix for topics and schema names, so it's important that it be explicitly set.